src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -127,6 +126,12 @@ private function getPersistableClasses(ObjectManager $manager): array
         return $persistableClasses;
     }
 
+    protected function setIdGenerator(ORMClassMetadataInfo $metadata): void
```

Comment:
you can remove the `@final` tag and comment as it's no longer true, I would however make the public method `persist` final as it does not have much sense to override it

Reply:
Removed, but did not make `persist()` final. I found a comment in some issue which encourages to override it because of missing entity ID issue.

Reply:
Fair, it's fine then